geomstats/utils.py

```diff
@@ -0,0 +1,8 @@
+import geomstats.backend as gs
```

Comment:
NIT: Could we give `utils` a more explicit name? Maybe `algebra.py` or `algebra_utils.py`? Or could this function go to `vectorization.py`?

Calling a file `utils.py` might lead us to put there everything that we do not know where to place, without forcing us to think about the repo's logic.

Reply:
That's a fair point. I had thought of adding it to vectorization, but at least judging by the function name it didn't really fit. I would prefer `algebra_utils` in that case.